core/templates/pages/exploration-editor-page/feedback-tab/feedback-tab.directive.ts

```diff
@@ -246,7 +246,7 @@ angular.module('oppia').directive('feedbackTab', [
               text: ''
             };
             ctrl.clearActiveThread();
-            ThreadDataService.fetchFeedbackStats();
+            ThreadDataService.getOpenThreadsCountAsync();
```

Comment:
Seems odd that the value that is presumably returned by this doesn't seem to be stored anywhere (and I don't see any callbacks).

Might be ok, just thought I'd flag it.

Reply:
Agreed, I'll try removing it and seeing if things pass without it.